distribution/lib/Standard/Database/0.0.0-dev/src/DB_Column.enso

```diff
@@ -327,13 +327,13 @@ type DB_Column
     make_op : Text -> Vector Text -> (Text | Nothing) -> (Any | Nothing) -> DB_Column
     make_op self op_kind operands new_name metadata=Nothing =
         checked_support = if self.connection.dialect.is_supported op_kind then True else
-            Error.throw (Unsupported_Database_Operation.Error "The operation "+op_kind+" is not supported by this backend.")
+            Error.throw (Unsupported_Database_Operation.Error op_kind)
         checked_support.if_not_error <|
             type_mapping = self.connection.dialect.get_type_mapping
             prepare_operand operand = case operand of
                 other_column : DB_Column ->
                     if Helpers.check_integrity self other_column then other_column.expression else
-                        Error.throw <| Unsupported_Database_Operation.Error "Cannot use columns coming from different contexts in one expression without a join."
+                        Error.throw <| Error "Cannot use columns coming from different contexts in one expression without a join."
```

Comment:
What does the bare `Error` mean here?

Reply:
It means I missed replacing it with the correct error type. And that we don't have a test for this line of code :)

Reply:
Good catch thank you

Reply:
I have fixed this line and added a ticket to add the tests https://github.com/enso-org/enso/issues/11176